src/components/Reserve/AddReserve.js

```diff
@@ -1,11 +1,81 @@
-import React, { Fragment } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { postBooking, setStatus } from '../../redux/bookings/bookings';
 
-function AddReserve() {
-  return (
-    <>
-      <div className="">AllProducts AddReserve</div>
-    </>
+const AddReserve = () => {
+  const [bookingDate, setBookingDate] = useState('');
+  const [bookingDateEnd, setBookingDateEnd] = useState('');
+
+  const { id } = useParams();
+  const idToUse = Number(id);
+  const { status } = useSelector((state) => state.bookings);
```

Comment:
The user is not able to access the addReserve page because of an error occurring in this line. Kindly remove it.